socorro/external/es/supersearch.py

```diff
@@ -422,6 +414,29 @@ def get(self, **kwargs):
 
                 search.aggs.bucket(f, agg_bucket)
 
+        # Create sub-sub-aggregations.
+        if params.get('_aggs.product.version'):
+            lvl1_field = 'product'
+            lvl2_field = 'version'
+
+            lvl1_bucket = self._get_fields_agg(lvl1_field, facets_size)
+            lvl2_bucket = self._get_fields_agg(lvl2_field, facets_size)
+
+            lvl1_bucket.bucket(
+                lvl2_field,
+                lvl2_bucket,
+            )
+
+            for param in params['_aggs.product.version']:
+                self._add_second_level_aggs(
+                    param,
+                    lvl2_bucket,
+                    facets_size,
+                    histogram_intervals,
+                )
+
+            search.aggs.bucket(lvl1_field, lvl1_bucket)
```

Comment:
I don't understand why you hardcoded this. I can see why you hardcoded the `_aggs.product.version` in the `SearchBase` class but these lines above can all be done with a looping over the keys. 

```python
for key in params:
    if key.startswith('_aggs.'):
        lvl1, lvl2 = key.split('.')[1:]
        ...
```